iocore/net/Net.cc

```diff
@@ -52,6 +56,20 @@ configure_net()
   // These are not reloadable
   REC_ReadConfigInteger(net_event_period, "proxy.config.net.event_period");
   REC_ReadConfigInteger(net_accept_period, "proxy.config.net.accept_period");
+
+  // This is kinda fugly, but better than it was before (on every connection in and out)
+  // Note that these would need to be ats_free()'d if we ever want to clean that up, but
+  // we have no good way of dealing with that on such globals I think?
```

Comment:
If that's a problem then make the globals `std::string` instances. Then it gets duplicated once, and cleaned up.